benchmarks/TF/CMakeLists.txt

```diff
@@ -16,18 +16,18 @@
 #                                                                              #
 ################################################################################
 
-set(MINILM_L12_H384_UNCASED_INT32_MODULE
+set(MINILM_L12_H384_UNCASED_SEQLEN128_INT32_MODULE
   NAME
-    "MiniLML12H384Uncased"
+    "MiniLML12H384UncasedSeqlen128"
```

Comment:
This actually should be a part of the `TAGS` I think.

Reply:
Done.